applications/templates/applications/page.html

```diff
@@ -40,6 +40,10 @@ <h1 class="display-4">{{ title }}</h1>
 <article class="container" id="applicationForm">
   <div class="row">
     <div class="col-lg-8">
+      <div class="progress mb-4">
+        <div class="progress-bar" role="progressbar" style="width: {{ progress_pc}}%" aria-valuenow="{{ progress_pc}}" aria-valuemin="0" aria-valuemax="100">{{ progress_pc }}%</div>
```

Comment:
Can you break the attributes down into multiple lines?  Makes it easier to scan down them!